packages/lesswrong/components/ea-forum/auth/EALoginPopover.tsx

```diff
@@ -421,95 +458,99 @@ export const EALoginPopover = ({action: action_, setAction: setAction_, facebook
-          </div>
-          {isSignup
-            ? (
-              <div className={classes.switchPrompt}>
-                Already have an account?{" "}
-                <a
-                  onClick={onLinkToLogin}
-                  className={classes.switchPromptButton}
-                >
-                  Login
-                </a>
+
+          {!showFacebookWarning && (
+            <>
+              <div className={classes.orContainer}>
+                <span className={classes.orHr} />OR<span className={classes.orHr} />
               </div>
-            )
-            : (
-              <div className={classes.switchPrompt}>
-                Don't have an account?{" "}
-                <a
-                  onClick={onLinkToSignup}
-                  className={classes.switchPromptButton}
-                >
-                  Sign up
-                </a>
+              <div className={classes.socialContainer}>
+                {googleEnabled && (
+                  <EAButton
+                    style="grey"
+                    variant="outlined"
+                    onClick={onClickGoogle}
+                    className={classNames(classes.button, classes.socialButton)}
+                  >
+                    <img src={links.googleLogo} alt="google logo" /> Continue with Google
+                  </EAButton>
+                )}
+                {facebookEnabled && !isSignup && (
+                  <EAButton
+                    style="grey"
+                    variant="outlined"
+                    onClick={onClickFacebook}
+                    className={classNames(classes.button, classes.socialButton)}
+                  >
+                    <FacebookIcon />
+                    Continue with Facebook
+                  </EAButton>
+                )}
               </div>
-            )
-          }
+
+              {isSignup ? (
+                <div className={classes.switchPrompt}>
+                  Already have an account?{" "}
+                  <a onClick={onLinkToLogin} className={classes.switchPromptButton}>
+                    Login
+                  </a>
+                </div>
+              ) : (
+                <div className={classes.switchPrompt}>
+                  Don't have an account?{" "}
+                  <a onClick={onLinkToSignup} className={classes.switchPromptButton}>
+                    Sign up
+                  </a>
+                </div>
+              )}
+            </>
+          )}
         </div>
-        {isSignup && <div className={classes.finePrint}>
-          By creating an{" " + forumTitleSetting.get() + " "}account, you agree to the{" "}
-          <Link to={links.terms} target="_blank" rel="noopener noreferrer">
-            Terms of Use
-          </Link> and{" "}
-          <Link to={links.privacy} target="_blank" rel="noopener noreferrer">
-            Privacy Policy
-          </Link>.
-        </div>}
+
+        {isSignup && !showFacebookWarning && (
```

Comment:
Not sure why it's necessary to test for !showFacebookWarning here

Reply:
Since the Facebook button has been removed from the signup form it's not actually necessary, but in principle isSignup=true, showFacebookWarning=true is an allowed state